arts-main/arts_core/models.py

```diff
@@ -101,6 +102,8 @@ class Module(models.Model):
     source = models.ForeignKey(
         'File', on_delete=models.PROTECT, blank=True, null=True,
         help_text="Source file identifier (for profile tracking)")
+    wasm = models.CharField(max_length=65535, default="", blank=True,
```

Comment:
CharField with very long max_length should be TextField instead

Reply:
Fixed